systemtest/src/test/java/io/strimzi/systemtest/operators/RecoveryST.java

```diff
@@ -228,6 +235,42 @@ void testRecoveryFromKafkaBridgeMetricsConfigDeletion() {
         timeMeasuringSystem.stopOperation(timeMeasuringSystem.getOperationID());
     }
 
+    @Test
+    void testRecoveryFromImpossibleMemoryRequest() {
```

Comment:
Add javadoc here and to `testKafkaPodPending` to explain the difference between the tests.